### PR TITLE
preparing v1.8 branch

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -27,124 +27,8 @@
...
-/test/Makefile* @cilium/ci @cilium/build
-/tests/ @cilium/ci
-Vagrantfile @cilium/ci
-/Vagrantfile @cilium/contributing
-/go.sum @cilium/vendor
-/go.mod @cilium/vendor
-/vagrant_box_defaults.rb @cilium/ci
-/vendor/ @cilium/vendor
+api/ @cilium/api
+pkg/apisocket/ @cilium/api
```
Comment (joestringer) [inline CODEOWNERS]: What is `pkg/apisocket`?

File: CODEOWNERS [hunk 2]
```diff
@@ -27,124 +27,8 @@
...
-/test/bpf/ @cilium/ci @cilium/bpf
-/test/Makefile* @cilium/ci @cilium/build
-/tests/ @cilium/ci
-Vagrantfile @cilium/ci
-/Vagrantfile @cilium/contributing
-/go.sum @cilium/vendor
-/go.mod @cilium/vendor
-/vagrant_box_defaults.rb @cilium/ci
-/vendor/ @cilium/vendor
+api/ @cilium/api
```
Comment (joestringer) [inline CODEOWNERS]: Should all of these start with `/`?

**Review by joestringer — CHANGES_REQUESTED** [review]
Couple minor nits, do we need to fix up `VERSION` as well?

File: CODEOWNERS (hunk 1 above)
Comment (aanm) [reply CODEOWNERS]: I copy paste from the docs :/

**Comment by aanm** [comment]
That's done here https://github.com/cilium/cilium/pull/11783

File: CODEOWNERS [hunk 3]
```diff
@@ -28,123 +28,3 @@
...
-/examples/minikube/ @cilium/kubernetes
-*.Jenkinsfile @cilium/ci
-/hubble-relay/ @cilium/hubble
-/hubble-relay.Dockerfile @cilium/hubble
-/install/kubernetes/ @cilium/kubernetes @cilium/docs @cilium/helm
-jenkinsfiles @cilium/ci
-Jenkinsfile.nightly @cilium/ci
-/operator/ @cilium/operator
-/pkg/annotation @cilium/kubernetes
-/pkg/api/ @cilium/api
```
Comment (joestringer) [inline CODEOWNERS:77]: The v1.7 equivalent commit  fd3b9ace2b278af74629d7767d71ae112b75b8cf leaves all of the @cilium/api codeowners in place, we should probably do the same here?

**Review by joestringer — APPROVED** [review]

**Pull request merged**